### ISSUE TITLE
Error: App does not seem to be open 

### ISSUE THREAD

**Issue by anjaw3f** [Zondax/ledger-kusama#49]
We have a couple of users reporting this error when trying to attach their ledger: 

![image](https://user-images.githubusercontent.com/70277946/103340041-c28f1a80-4a8b-11eb-83a3-2f3f220c2777.png)

Messages from two users: 

"My Ledger app is updated. I can’t open dot app on my ledger Live X. It is open for a moment and the it close. I have google chrome. I try with Brave browser but it is the same problem. It’s look like my Dot app on Ledger doesn’t work."

"I have downloaded the Polkadot App... I clicked attach ledger via web usb... I go to accounts add ledger... it says Ledger Nano X Paired... and then the circle never stops moving an the ledger never loads.... then it says app does not seem to be open, but it is.

**Comment by jleni** [comment]
This seems a duplicate of this issue.
https://github.com/Zondax/ledger-polkadot/issues/30#issuecomment-733160034

It seems that people do not open the app before clicking save. 

> I would suggest the following workflow (a slightly different from what you did)
> 1. Enter the PIN
> 2. Open the Polkadot App
> 2. Click Add account in the UI
> As far as I know, when you click add account the request is immediately sent to the app. If you don't have the app already opened, then it will fail. You need to open the device before clicking "Add Account".
>

A recommendation for polkadot.js would be to add a warning that the Ledger app NEEDS to be open before clicking save.

**Comment by jleni** [comment]
Closing, please reopen and let us know if the solution was not adequate.
